data/presets/building/annexe.json

```diff
@@ -0,0 +1,16 @@
+{
+    "icon": "maki-home",
+    "geometry": [
+        "area"
+    ],
+    "tags": {
+        "building": "annexe"
+    },
+    "terms": [
+        "annexe",
+        "adu",
+        "granny"
+    ],
+    "matchScore": 0.5,
+    "name": "Annexe House"
```

Comment:
The `name` needs to be in American English, but “annexe” is completely unknown to American English speakers. In California, the formal term is “Accessory Dwelling Unit” (hence ADU), but this might be unknown in other parts of the state, compared to something more crass like “in-law unit” or “granny suite”. I suggest asking folks in the #language channel of OSMUS Slack for second opinions.

Reply:
I'll use "Accessory Dwelling Unit" for now. Translators can choose the best term for their region.